### PR TITLE
add json output type

### PR DESCRIPTION
**Pull request by friehl**

### CODE REVIEW

File: src/sqlquest.coffee [hunk 1]
```diff
@@ -53,6 +53,7 @@ opts = nomnom()
     help: 'Print the runtime of each query'
   )
   .option 'quest', position: 0, help: 'Which quest to embark on!'
+  .option 'output', abbr: 'o', help: 'Which output format.'
```
Comment (avram) [inline src/sqlquest.coffee]: Should list the options

File: src/quest.coffee [hunk 2]
```diff
@@ -226,6 +227,15 @@ class Quest
     else
       console.log "Nothing to output.".gray
 
+  jsonify: (sqlResult, options) ->
+    console.log JSON.stringify row, 4 for row in sqlResult.rows
```
Comment (Raynes) [inline src/quest.coffee]: Just call `JSON.stringify` on `sqlResult.rows` which will output it as a json list of objects.

File: src/quest.coffee [hunk 3]
```diff
@@ -226,6 +227,15 @@ class Quest
...
+
+  outputType: (sqlResult, options) ->
```
Comment (Raynes) [inline src/quest.coffee]: We're not using the 'options' argument. Also, let's name this 'outputRows' or 'output'. Any particular reason you decided on outputType?

File: src/sqlquest.coffee [hunk 4]
```diff
@@ -53,6 +53,7 @@ opts = nomnom()
     help: 'Print the runtime of each query'
   )
   .option 'quest', position: 0, help: 'Which quest to embark on!'
+  .option 'output', abbr: 'o', help: 'Options are: table or json'
```
Comment (Raynes) [inline src/sqlquest.coffee]: Default to table.

**Comment by Raynes** [comment]
Great work!

**Pull request merged**